run_sorting_hat_test_data.sh

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env bash
 
 GTF=$( pwd )/test-data/gff
-DELETE_PREV_OUTPUT=False
+FORCEALL=$true
+RESTART_TIMES=1
 SKIP_STEPS=$( pwd )/step_skip.yaml
+CHECK_SPECIES=$( pwd )/Bulk_recalculations_check_species_names.yaml
 BIOENTITIES_PROPERTIES=$( pwd )/test-data/bioentity_properties
 SORTING_HAT=$( pwd )/Snakefile-sorting-hat
 LOG_HANDLER=$( pwd )/log_handler.py
 SN_CONDA_PREFIX=${SN_CONDA_PREFIX:-$( pwd )/conda_installs}
+if [ $FORCEALL ]
+then
+  PROFILE_LINE="--profile profilename --restart-times "$RESTART_TIMES
+else
+  PROFILE_LINE="--profile profilename --forceall --restart-times "$RESTART_TIMES
+fi
```

Comment:
I wouldn't mix all these things together here... if forceall should only change a $FORCEALL_LINE or something alike.

Reply:
See ddce3e9